src/client/src/orangehrmRecruitmentPlugin/pages/ShortlistCandidateScreen.vue

```diff
@@ -0,0 +1,183 @@
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <div class="orangehrm-background-container orangehrm-save-candidate-page">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6" class="orangehrm-main-title">
+        Shortlist Candidate
+      </oxd-text>
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Candidate"
+                disabled
+                :value="shortlist.candidate"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Vacancy"
+                disabled
+                :value="shortlist.vacancy"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Hiring Manager"
+                disabled
+                :value="shortlist.manager"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Current Status"
+                disabled
+                :value="shortlist.status"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item class="orangehrm-save-candidate-page-full-width">
+              <oxd-input-field
+                v-model="shortlist.note"
+                label="notes"
+                type="textarea"
+                :rules="rules.note"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button display-type="ghost" :label="$t('general.back')" />
+          <submit-button label="Shortlist" />
```

Comment:
Need space between buttons. please check other places and use same classes

Reply:
space was already in there